src/reactviews/pages/QueryResult/table/plugins/copyKeybind.plugin.ts

```diff
@@ -48,18 +48,9 @@ export class CopyKeybind<T extends Slick.SlickData> implements Slick.Plugin<T> {
 
     private async handleKeyDown(e: KeyboardEvent): Promise<void> {
         let handled = false;
-        let platform = await this._qrContext.extensionRpc.sendRequest(GetPlatformRequest.type);
```

Comment:
Removed this as it is async and it doesn't work when we want to stop default behavior or stop propagation for the event. 
while we are not doing that here,  I don't want to give a wrong idea to someone copying from this code.